python/src/trezorlib/cli/trezorctl.py

```diff
@@ -250,6 +250,16 @@ def usb_reset():
     WebUsbTransport.enumerate(usb_reset=True)
 
 
+@cli.command(help="Show text on Trezor display.")
```

Comment:
please create a new file `trezorlib/cli/debug.py` (take inspiration from e.g. `crypto.py` which is reasonably short) and move this to it.

also move the help text to a docstring (see other commands)

also it would be nice to have a documented list of possible values for `icon`. The most straightforward way to accomplish that is to do a `ChoiceType` for the argument type; see `get_address` in `btc.py`.

Instead of enumerating the choices by hand, you could use code like this:
```python
from ..messages import DebugShowTextIcon

ICONS = {
    key: getattr(DebugShowTextIcon, key)
    for key in dir(DebugShowTextIcon)
    if not key.startswith("__")
}
```

then use `type=ChoiceType(ICONS)` for the `icon` argument